index.html

```diff
@@ -42,7 +42,7 @@
           <a class="desktop-link" href="#">Contact</a>
```

Comment:
- [ ] It might be a good idea to update your portfolio page title because it currently is "Portfolio Mike" and the correct form should be "Mike's Portfolio". I think this way could be more understandable for your users :wink: